package.json

```diff
@@ -10,8 +10,8 @@
   "main": "app/index.js",
   "scripts": {
     "build": "npm run build:css",
-    "build:css": "node-sass --output=app/static/stylesheets --output-style=expanded app/static/stylesheets",
-    "build:css-watch": "node-sass --output=app/static/stylesheets --output-style=expanded --source-map true --watch app/static/stylesheets",
+    "build:css": "sass --load-path=./ app/static/stylesheets",
+    "build:css-watch": "npm run build:css -- --watch",
```

Comment:
Rather than duplicating the options for `build:css` I just run the task add pass the `--watch` flag.